### PR TITLE
Some CSS improvements

### PR DESCRIPTION
**Pull request by Bl4Cc4t**
Most of them are related to the layout of the file.
However, there are two functional improvements in `lrr.css` at the very bottom.
https://github.com/Difegue/LANraragi/blob/1d6752088f8168c9f1ef4ca56020c20327a6aef5/public/css/lrr.css#L432-L438
This was done a long time ago and I initially wanted to change the other CSS files too, so I put this on hold :V

### CODE REVIEW

File: public/css/lrr.css [hunk 1]
```diff
@@ -1,363 +1,438 @@
...
+  width: 60%;
 }
 
 .if a {
-    margin-right: 10px;
+  margin-right: 10px;
+}
+
+li {
+  list-style: none;
```
Comment (Difegue) [inline public/css/lrr.css]: This is new afaik -- Do you remember why you put it there?

File: tools/build/homebrew/lanraragi [hunk 2]
```diff
@@ -20,6 +20,9 @@ fi
 if [[ ${LRR_DATA_DIRECTORY} == "" ]]; then
```
Comment (Difegue) [inline tools/build/homebrew/lanraragi]: We did merge this already, so it's weird to see it pop up again -- A rebase of the branch would be most welcome.

File: tools/build/homebrew/redis.conf [hunk 3]
```diff
@@ -35,6 +35,14 @@
 # include /path/to/local.conf
```
Comment (Difegue) [inline tools/build/homebrew/redis.conf]: ditto

File: public/css/lrr.css (hunk 1 above)
Comment (Bl4Cc4t) [reply public/css/lrr.css]: Yes, that is to get rid of the little dots on `/logs`
Bildschirmfoto 2020-01-23 um 21 08 50

**Comment by Bl4Cc4t** [comment]
I didn’t even saw the other two files, forgot to merge beforehand. Everything should be fine now :)

File: tools/build/homebrew/lanraragi (hunk 2 above)
Comment (Bl4Cc4t) [reply tools/build/homebrew/lanraragi]: I merged it again

**Comment by CirnoT** [comment]
Do not merge PR branches, this creates a mess in history - rebase them.

**Comment by Difegue** [comment]
Well, luckily squashing is a thing, so I won't end up with duplicate commits in the dev tree.  

Thanks for the css fixes!

**Pull request merged**